analytics-test.rb

```diff
@@ -0,0 +1,29 @@
+require 'FileUtils'
+
+puts "Enter the filetype you want to search (rb, erb, md, or html)"
+filetype = gets.chomp.to_s
+
+puts "Processing #{filetype} files recursively in #{Dir.pwd} with Rorybot"
+`rorybot **/*.#{filetype} | tee rorybot_output.txt`
+
+input_file = File.open('rorybot_output.txt', 'r')
+output_file = File.open('frequency.csv', 'w+')
+warning_message = /(?:warning\s{2,}?)(.+?)(?=\s{2,})/
+frequency = Hash.new(0)
+
+puts "Counting warning messages"
+input_file.read.scan(warning_message) do |word|
```

Comment:
Ah, I think I misread this. That said, `frequency[word] = frequency[word] + 1` should work as `frequency[word] += 1`